chat-widget/src/plugins/newMessageEventHandler.ts

```diff
@@ -20,15 +20,13 @@ export const createOnNewAdapterActivityHandler = (chatId: string, userId: string
     const raiseMessageEvent = (activity: IActivity) => {
         if (activity?.type === Constants.message) {
             const payload = {
+                // To identify hidden contents vs no content
                 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-                text: (activity as any)?.text,
-                id: activity?.id,
+                text: (activity as any)?.text?.length >= 1 ? "*contents hidden*" : "",
```

Comment:
Why >= 1 not >1?

Reply:
This is for single char message validation